File: examples/fetchSongInfo.js
// Function to fetch data from the API
async function fetchSongInfo(artist_name, song_name) {
    const url = `https://yutify.onrender.com/api/${artist_name}:${song_name}`;

    const response = await fetch(url);
    const data = await response.json();

    // Do anything with `data`, here log to console
    console.log(data);
}

// Call the function with arguments
fetchSongInfo("Artist Name", "Song Name");

File: examples/fetch_song_info.py
import requests


def fetch_song_info(artist_name, song_name):
    url = f"https://yutify.onrender.com/api/{artist_name}:{song_name}"

    response = requests.get(url)
    data = response.json()

    # Do anything with `data`, here print in the terminal
    print(data)


# Call the function with arguments
fetch_song_info("Artist Name", "song Name")
